New tutorial: how to protect a REST endpoint

gleybersonandrade opened the pull request:
Now that kytos will support secure endpoints, we needed to create a tutorial teaching users how to create a NApp with protected endpoints. This commit  has a tutorial to solve this demand.

hdiogenes commented on tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst:
```diff
@@ -0,0 +1,309 @@
+Introduction
+************
+
+Beyond the events, one of the methods to communicate with your applications are
```
```suggestion
Besides events, another method to communicate with your NApp is
```

hdiogenes commented on tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst:
```diff
@@ -0,0 +1,309 @@
+************
+
+Beyond the events, one of the methods to communicate with your applications are
+through REST endpoints. These endpoints can be public and private, this last is
```
```suggestion
through REST endpoints. These endpoints can be public or private (note that
```

hdiogenes commented on tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst:
```diff
@@ -0,0 +1,309 @@
+
+Beyond the events, one of the methods to communicate with your applications are
+through REST endpoints. These endpoints can be public and private, this last is
+still under development.
```
```suggestion
the latter is still under development).
```

hdiogenes commented on tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst:
```diff
@@ -0,0 +1,313 @@
+
+Congratulations! You have completed the tutorial and explored the creation of REST
+endpoints using different request methods like POST, PATCH and DELETE. One tip is
+that you can use a REST client named Postman to test your application.
```
Please add a link to the Postman app page.

hdiogenes commented on tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst:
```diff
@@ -0,0 +1,313 @@
+
+.. code-block:: console
+
+    GET http://0.0.0.0:8181/api/<username>/<napp_name>/name
```
curl

hdiogenes commented on tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst:
```diff
@@ -0,0 +1,313 @@
+
+.. code-block:: console
+
+    GET http://0.0.0.0:8181/api/<username>/<napp_name>/name
```
Please add examples directly as curl commands on the shell.

hdiogenes commented on tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst:
```diff
@@ -0,0 +1,313 @@
+The creation of a new endpoint is made by the use of the `@rest` decorator,
+and it can be optionally protected using the `@authenticated`
+decorator. More information about this module can be found in
+the |protect_rest|_ and in the Auth documentation.
```
Is there another page with the Auth documentation whose link is missing from here? Please add link.

gleybersonandrade committed:
Create tutorial to teach users how to protect an endpoint

Now that kytos will support secure endpoints, we needed to create a tutorial teaching users how to create a NApp with protected endpoints. This commit  has a tutorial to solve this demand.
Update how_to_protect_a_napp_rest_endpoint.rst

Text review
Auto stash before rebase of "origin/new"

hdiogenes committed:
Update how_to_protect_a_napp_rest_endpoint.rst

hdiogenes committed:
Update how_to_protect_a_napp_rest_endpoint.rst

Link to Postman app
